dotnet-format/info.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: dotnet-format
+image: restyled/restyler-dotnet-format:v0.0.1
+command:
+  - dotnet-format
+  - --files
+arguments: []
+include:
+  - "**/*.cs"
+  - "**/*.vb"
+supports_arg_sep: false
+# It is supported, but only comma separated
```

Comment:
Could add a shim-script to accept path arguments and build the comma-separated invocation.